protected/migrations/m170821_053800_cera_clean.php

```diff
@@ -72,14 +49,11 @@ public function safeUp()
         if (!$address->save()) {
             throw new CDbException("Unable to save address");
         }
-        else {
-            $address->save();
-        }
 	}
 
 	public function safeDown()
 	{
-
+        echo "m170821_053800_cera_clean does not support migration down.\n";
```

Comment:
If down migration isn't supported, this function needs to return false so it throws an error when you attempt to do so.